metadata.json

```diff
@@ -28,6 +28,10 @@
     {
       "name": "puppet/archive",
       "version_requirement": ">= 1.0.0 < 8.0.0"
+    },
+    {
+      "name": "zend/zend_common",
+      "version_requirement": ">= 1.1.1 < 8.0.0"
```

Comment:
I think we should leave this out of `metadata.json` since it makes `zend/zend_common` a hard requirement even when ZendPHP is not being used. Instead you can explain that `zend/zend_common` is needed for ZendPHP.

Reply:
Fair enough. In that case, I can just remove metadata.json from the commit, which makes it easy!

Reply:
Please add some lines on README explaining that in order to use ZendPHP they need `zend/zend_common` puppet module, else people will be confused why their catalog is not compiling 😄

Reply:
Done, @root-expert :smile: